src/navigation/components/usersNavigation.ts

```diff
@@ -0,0 +1,11 @@
+import {createStackNavigator} from 'react-navigation';
+
+import Users from '../../screens/Users/components/Users';
```

Comment:
pass is wrong. Wee need to have an index file for our modules/screens. Make it '../../screens/Users/Users'